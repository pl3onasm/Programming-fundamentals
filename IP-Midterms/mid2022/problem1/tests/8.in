08:35:52 22:03:22
