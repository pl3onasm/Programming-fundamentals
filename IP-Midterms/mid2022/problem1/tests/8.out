13:27:30
